subsys/dfu/Kconfig

```diff
@@ -103,3 +95,12 @@ config UPDATEABLE_IMAGE_NUMBER
 endif
 
 endif # IMG_MANAGER
+
+config MCUBOOT_UPDATE_FOOTER_SIZE
```

Comment:
Not acceptable, the dependency is for `MCUBOOT_IMG_MANAGER` not just if MCUboot is enabled, see https://github.com/zephyrproject-rtos/zephyr/pull/90721